test/controllers/partners/invoice_files_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+require "webmock/minitest"
+
+class Partners::InvoicesControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  describe '#create' do
+    before do
+      partner = partners(:default_partner)
+      sign_in partner
+      Invoice.destroy_all
```

Comment:
Do we need to destroy all in the tests ? 💥

Reply:
I'd rather have a clean slate each test case, so we can ensure that there is a correct state during the test 😄

Reply:
👍